lib/src/header_parser/utils.dart

```diff
@@ -236,6 +236,10 @@ extension CXTypeExt on clang_types.CXType {
     return clang.clang_getTypeKindSpelling(kind()).toStringAndDispose();
   }
 
+  int align() {
```

Comment:
`alignment()` in Dart we don't really abbreviate method names. And this looks up the alignment rather than aligning something.

You could even do `int get alignement =>` if you want to be fancy. 😄

Reply:
Updated `align()` to -> `alignment()`

> You could even do int get alignment...

Yeah, I thought that too, but then it wouldn't be consistent with the code earlier.

Reply:
If you'd like, you can always send a refactoring PR to make it more pretty. I don't have a very strong opinion about this. :)

Reply:
Yeah, I might do that someday. There's a lot I think which can be improved.